### DISCUSSION TITLE
Release of v3.4.5

### DISCUSSION THREAD

**Discussion by KasnocknDave** [argoproj/argo-workflows#10461] [Q&A] [answered]
Hello everyone :wave: 

I wanted to ask when/if it is already planned to release a fix version v3.4.5.
There has been a pretty big pause in releases and we're currently waiting on the rollout for a go-git fix.
Described in https://github.com/argoproj/argo-workflows/issues/9613.

Thanks for the info in advance!

**Comment by agilgur5** [comment, marked as answer]
This was fixed in #9982 and released as part of [v3.4.5](https://github.com/argoproj/argo-workflows/releases/tag/v3.4.5) on Feb 7th, a few days after this discussion was created.
